pkg/operator/reports.go

```diff
@@ -486,7 +486,7 @@ func (op *Reporting) runReport(logger log.FieldLogger, report *metering.Report)
 	var runningMsg, runningReason string
 	if report.Spec.RunImmediately {
 		runningReason = meteringUtil.RunImmediatelyReason
-		runningMsg = fmt.Sprintf("Report %s scheduled: runImmediately=true bypassing reporting period [%s to %s].", report.Name, reportPeriod.periodStart, reportPeriod.periodEnd)
+		runningMsg = fmt.Sprintf("Report %s scheduled: runImmediately=true and reporting period [%s to %s].", report.Name, reportPeriod.periodStart, reportPeriod.periodEnd)
```

Comment:
Now that I've had some time to think about it, "bypassing" might be a decent description. We may just want to take it a step further and log something like "bypassing validation checks from reporting period ...". WDYT?

Reply:
But setting it true doesn't ignore, or bypass, the reportingEnd marker though. It bypasses `spec.schedule` values and I think the logging as changed reflects that and shows what reporting period is in use. The phrasing, as it was, continues the confusion that the reportingEnd might not be needed, IMHO.

Reply:
Hmm okay, that's a fair point. Maybe the best solution at this point is to hold off on making any more updates to this logging message.